include/ada/helpers.h

```diff
@@ -11,7 +11,6 @@
 namespace ada::helpers {
 
   ada_really_inline std::optional<std::string_view> prune_fragment(std::string_view& input) noexcept;
-  std::optional<uint16_t> get_port(std::string_view input) noexcept;
   std::optional<uint16_t> parse_port(std::string_view input,
```

Comment:
It seems parse port is not used anymore, can we remove it?

Reply:
Removed.